tools/server-spring/src/main/java/org/eclipse/rdf4j/http/server/repository/transaction/Transaction.java

```diff
@@ -472,28 +473,40 @@ public WildcardRDFRemover(RepositoryConnection conn) {
 
 		@Override
 		public void handleStatement(Statement st) throws RDFHandlerException {
+			System.out.println(st);
+			Thread.dumpStack();
 			Resource subject = SESAME.WILDCARD.equals(st.getSubject()) ? null : st.getSubject();
 			IRI predicate = SESAME.WILDCARD.equals(st.getPredicate()) ? null : st.getPredicate();
 			Value object = SESAME.WILDCARD.equals(st.getObject()) ? null : st.getObject();
 
-			// use the RepositoryConnection.clear operation if we're removing
-			// all statements
-			final boolean clearAllTriples = subject == null && predicate == null && object == null;
+			// use the RepositoryConnection.clear operation if we're removing all statements
+			boolean clearAllTriples = subject == null && predicate == null && object == null;
```

Comment:
It's not really that important but is there a reason you remove final keywords on these local vars? I tend to deliberately put these in as it clarifies intent that these are constants.

Reply:
For short methods I feel that they add clutter to the code. I usually only use constants for class fields and not for code within a method. If the method code is long and hard to understand then it can help to add a final modifier when there are multiple variables where some should not be modified while others are intended to be modified later in the code.

In this case there are also 3 variables above this line that should have be declared as final, if we wanted to be concise.

Reply:
Java could do with a `val` keyword...